src/main/java/io/redisearch/client/Client.java

```diff
@@ -21,6 +21,7 @@
     private final String indexName;
     private final byte[] endocdedIndexName;
     private final Pool<Jedis> pool;
+    private Jedis jedis;
```

Comment:
1. This is not handled in `close()`.
2. Why not `final`?

Reply:
Done